Source/Core/DolphinQt2/QtUtils/QInputDialog.cpp

```diff
@@ -0,0 +1,24 @@
+// Copyright 2017 Dolphin Emulator Project
+// Licensed under GPLv2+
+// Refer to the license.txt file included.
+
+#include "DolphinQt2/QtUtils/QInputDialog.h"
+
+#include "Core/HotkeyManager.h"
+
+QInputDialog::QInputDialog(QWidget* parent) : QDialog(parent)
```

Comment:
Also, about the class name:
What would you suggest? ``BlockHotkeyDialog`` or something like that?